### PR TITLE
feat: initial routing

### LINKED ISSUE #5: Setup initial routing for dependency formatter

### PR DESCRIPTION
**Pull request by sashaqred**
Closes #5

### CODE REVIEW

File: src/App/routes/AppRouter.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+export function AppRouter() {
+  return (
+    <BrowserRouter>
+      <Switch>
+        <Redirect exact path="/" to="/dependency-formatter" />
+        <Route path="/dependency-formatter">
+          <DependencyFormatter />
+        </Route>
+        <Redirect path="*" to="/dependency-formatter" />
```
Comment (Roma36) [inline src/App/routes/AppRouter.tsx]: why do you need first redirect if this one covers both cases?

File: src/DependencyFormatter/containers/DependencyFormatter/DependencyFormatter.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <Switch>
+      <Redirect exact path={path} to={`${path}/upload`} />
+      <Route path={`${path}/upload`}>
+        <FileSelector onFileLoaded={uploadCallback} />
+      </Route>
+      <Route exact path={`${path}/format`}>
+        <h1>Format uploaded file</h1>
+      </Route>
+      <Redirect path={`${path}/*`} to={`${path}/upload`} />
```
Comment (Roma36) [inline src/DependencyFormatter/containers/DependencyFormatter/DependencyFormatter.tsx]: same question: why 2 redirects?

**Review by Roma36 — CHANGES_REQUESTED** [review]

File: src/App/routes/AppRouter.tsx (hunk 1 above)
Comment (sashaqred) [reply src/App/routes/AppRouter.tsx]: The first is redundant. Will fix

File: src/DependencyFormatter/containers/DependencyFormatter/DependencyFormatter.tsx (hunk 2 above)
Comment (sashaqred) [reply src/DependencyFormatter/containers/DependencyFormatter/DependencyFormatter.tsx]: Here was an issue with redirects. If we come from '/not-existed-path', app'll be redirected to 'dependency-formatter' without the following redirect to '/dependency-formatter/upload'.
But I found the fix.

**Review by Roma36 — APPROVED** [review]

**Pull request merged**